app/assets/javascripts/live_search.js

```diff
@@ -150,6 +150,16 @@
         });
       });
     }
+
+    var $results = $('.finder-results');
+    if ($results.length > 0) {
+      if ($results.find('.document--top .document__link').length == 1) {
+        var trackingAction = $results.find('.document--top .document__link').attr('data-track-action');
+        trackingAction += 'r';
+        $results.find('.document--top .document__link').attr('data-track-action', trackingAction);
```

Comment:
if you refactor the `.document__link` in the scss file then you could refactor this to `.find('.document__link--top')` or better yet use the cached results